belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/api/InputValidationIssues.java

```diff
@@ -67,7 +69,7 @@ public static InputValidationIssue schemaViolation(InEnum in, String name, Objec
 
     public static InputValidationIssue unknownInput(InEnum in, String name, Object value) {
         return new InputValidationIssue(ISSUE_TYPE_UNKNOWN_INPUT, "Unknown input")
-                .detail(String.format("Input %s is unknown", name))
+                .detail(I18N.getLocalizedString("unknownInput.detail", name))
```

Comment:
Would it be interesting to add a `localizedDetail` method, similar to `I18N.getLocalizedDetail(...)`? Would make the code less verbose + remove duplication of ".detail" part of the bundle key.